data/human/free worlds 4 epilogue.txt

```diff
@@ -44,8 +44,8 @@ mission "FW Epilogue: Ijs and Katya"
 				goto end
 			
 			label game
-			`	"Whoops," says Ijs, "now you've done it, there goes the fourth wall."`
-			`	Katya laughs. "Well, there's already lots beyond human space to see, and I'm sure you can find more things to do out there as well as closer to home. If you're so eager to explore more of the galaxy, <first>, or to discover what new adventures it has to offer, maybe you should volunteer to help out with creating the stories to populate it. This is an open source game after all, and it relies on people like you to bring it life."`
+			`	"Whoops," says Ijs, "now you've done it. There goes the fourth wall."`
+			`	Katya laughs. "Well, there's already lots beyond human space to see, and I'm sure you can find more things to do out there as well as closer to home. If you're so eager to explore more of the galaxy, <first>, or to discover what new adventures it has to offer, maybe you should volunteer to help out with creating the stories that populate it. This is an open source game after all, and it relies on people like you to bring it to life."`
```

Comment:
This was fine for both changes.

Reply:
I'm fine with either the original or the changed version. If Derpy prefers the original, then I'd go with that.